### PR TITLE
Add the GPG key to the README

### PR DESCRIPTION
**Pull request by samuelcole**

### CODE REVIEW

**Review by antauth — APPROVED** [review]
Seems legit

**Review by dap — COMMENTED** [review]
Please add instructions for verifying the signature.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
+## Installation
+
+### WordPress Plugin Repository
+
+If you want to install the plugin on your WordPress site, you can download and install the latest version of the plugin from WordPress.org: [ActBlue Contributions](https://wordpress.org/plugins/actblue-contributions/).
+
+### GitHub Releases
+
+You can also download the latest version of the plugin from the [GitHub releases page](https://github.com/actblue/actblue_wordpress_plugin/releases). These releases are signed with the [ActBlue GPG key](#ActBlue-GPG-key).
+
```
Comment (dap) [inline README.md:14]: Please added example commands for importing the key and verifying the release signature. See https://tails.boum.org/install/expert/usb/index.en.html for example.

Also, add at least the PGP key fingerprint if not the full key details. See https://tails.boum.org/doc/about/openpgp_keys/index.en.html#index2h1 for example.

Also, it is a **PGP** key not a **GPG** key.

**Review by dap — COMMENTED** [review]
Added info on specific changes needed.

**Pull request merged**